tikv-client/src/main/java/com/pingcap/tikv/operation/transformer/Cast.java

```diff
@@ -42,7 +42,7 @@ public void set(Object value, Row row, int pos) {
     } else if (targetDataType instanceof RealType) {
       casted = castToDouble(value);
     } else {
-      throw new UnsupportedOperationException("only support cast to Long, Double and String");
```

Comment:
Is this covered by regression tests?
Also you might make a patch onto refactor branch.

Reply:
Included in DAGTestCases.